modules/navier_stokes/test/tests/finite_volume/ins/jeffery-hamel/tests

```diff
@@ -0,0 +1,12 @@
+[Tests]
+  issues = '#21133'
+  design = 'insfv.md'
+  [wedge_dirichlet]
+    type = 'Exodiff'
+    input = 'wedge_dirichlet_vol_PA.i'
+    exodiff = 'wedge_dirichlet_vol_PA_out.e'
```

Comment:
I'd modify this one too and remove the corresponding exodus from the gold folder.

Reply:
I think I did it.

Reply:
Thanks!